### DISCUSSION TITLE
touchpad

### DISCUSSION THREAD

**Discussion by AnthonyBAC** [mylinuxforwork/dotfiles#213] [General]
I have tried searching all the conf files to invert the toouchpad since by default when I go up it goes up and when I go down it goes down, but I would like it to be inverted, I have searched and I have not found anything, I think that I am the one who has not known how to search, is there anyone who can help me?

**Comment by mylinuxforwork** [comment]
Go to .config/hypr/conf/keyboard.conf

https://github.com/mylinuxforwork/dotfiles/blob/main/share/dotfiles/.config/hypr/conf/keyboard.conf

Set natural_scroll to true.

**Reply by AnthonyBAC** [reply]
that? hahah I entered that conf several times, and I never read that, my god, I'm too blind or I'm very clueless haha ​​it must be one of the two, thank you very much!
